origins_media/origins_media.module

```diff
@@ -336,3 +336,23 @@ function origins_media_media_library_file_upload_process($form, FormStateInterfa
 
   return $form;
 }
+
+/**
+ * Implements hook_preprocess_media_oembed_iframe().
+ */
+function origins_media_preprocess_media_oembed_iframe(&$variables) {
+  // If we're embedding a Youtube embed object, and there isn't already
+  // and rel=0 parameter... add a rel=0 query parameter to the HTML
+  // to show related videos from the same channel as the video, not videos
+  // based on the viewer's recommendations from their own viewing history.
+  $html = &$variables['media'];
+
+  $is_youtube_embed = preg_match('|youtube.com\/embed|', $html);
```

Comment:
Should we also check against https://youtu.be/ ?

Reply:
Agreed. Fortunately, core now has a patch which lets us check the provider name (not URL pattern) and has a test which we can borrow the preprocess code from to get the desired result.